lib/merkle-tree.js

```diff
@@ -44,11 +45,10 @@ export default class MerkleTree {
     }, []);
   }
 
-  combinedHash (first, second) {
+  combinedHash(first,second ) {
     if (!first) { return second; }
     if (!second) { return first; }
-
-    return sha3(this.sortAndConcat(first, second));
+    return Buffer.from(hexToBytes(soliditySha3({t: 'bytes', v: this.sortAndConcat(first,second).toString("hex")})))
```

Comment:
If I'm not mistakened, we can use the original `sha3()` from ethereum js util to make this deal with buffers only-- removing the conversions to hexes. Tho, it might be even more confusing to have two different sha functions being used. What do you think?

Reply:
I agree. I think your suggestion is fine

Reply:
@habdelra I think I misread this comment and made [this pr](https://github.com/cardstack/merkle-tree-payment-pool/pull/8)

Can you confirm that you meant to include this `Buffer.from(hexToBytes(soliditySha3({t: 'bytes', v: this.sortAndConcat(first,second).toString("hex")})))`.

If so, I will revert the refactor-part of the change in [this pr](https://github.com/cardstack/merkle-tree-payment-pool/pull/8).

Reply:
I’d like to try the keccak256(), I think it makes the code more easy to read with that approach.

Reply:
That said, I do not have strong feelings here. I leave it to your judgement. I think either approach is fine.

Reply:
Ok cool. I will leave the pr as it is since it makes the change to `keccak256`.